CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the APM Server
+
+The APM Server is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code.
+
+You can get in touch with us through [Discuss](https://discuss.elastic.co/c/apm), feedback and ideas are always welcome.
+
+## Code contributions
+
+If you have a bug fix or new feature that you would like to contribute with, please find or open an issue about it first and explain briefly your approach.
```

Comment:
"briefly" should be at the end or before before explain. Alternatively we use this whole thing directly Elasticsearch:

> If you have a bugfix or new feature that you would like to contribute to APM Server, please find or open an issue about it first. Talk about what you would like to do. It may be that somebody is already working on it, or that there are particular issues that you should know about before implementing the change.

> We enjoy working with contributors to get their code accepted. There are many approaches to fixing a problem and it is important to find the best approach before writing too much code.

Reply:
changed 51194326238229cf494a46175142ade82ff63cc9